app/controllers/incomes_controller.rb

```diff
@@ -17,24 +17,27 @@ def show
   # GET /incomes/new
   def new
     @round = current_game.round
-    @countries = Game::COUNTRIES
+    @teams = Team.all
     @income = Income.new
   end
 
   # GET /incomes/1/edit
   def edit
     @round = @income.round
-    @countries = Game::COUNTRIES
+    @teams = Team.all
   end
 
   # POST /incomes
   # POST /incomes.json
   def create
-    @income = Income.new(income_params)
+    teamId = income_params['team'].to_i
+    new_params = income_params.except('team')
+    @income = Income.new(new_params)
+    @income.team = Team.find(teamId)
 
     respond_to do |format|
       if @income.save
-        current_game.push(@income)
+     #   current_game.push(@income)
```

Comment:
I'm not super sure what this is actually doing, but it was breaking. I feel like we should simply set the `game_id` when we create the income record? But it's also possible that I'm definitely missing something here.